modules/ocf_mirrors/files/healthcheck

```diff
@@ -19,9 +20,29 @@ import requests
 Mirror = namedtuple('Mirror', ['url', 'updated_at'])
 
 
-def get_updated(mirror_url):
-    """Find the time the host was last updated.
+def update_func(healthcheck):
+    if healthcheck == 'debian':
+        return get_updated_debian
+    elif healthcheck == 'ts':
+        return get_updated_ts
+    else:
+        return get_updated_debian
+
+
+def get_updated_ts(mirror_url):
+    """Find the time the host was last synced"""
+
+    req = requests.get(mirror_url)
+    req.raise_for_status()
+    try:
+        return datetime.utcfromtimestamp(int(req.text.split()[0]))
+    except (TypeError, ValueError):
+        # UTC Timestamp ... parrot
```

Comment:
Parrot?

Reply:
Oh, I see, it just tries to parse it however it can, is this what was needed for parrot's health check? If so I'd make a new check for it instead of doing this, because that seems safer, then we'll know if something is going wrong more easily. Otherwise I could see a mirror returning something that maybe parses into a date way in the past or something, and then we don't know that it isn't syncing correctly any more.